libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/WPHtmlTagHandler.java

```diff
@@ -7,15 +7,16 @@
 
 import org.xml.sax.XMLReader;
 
-import java.util.Vector;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Handle tags that the Html class doesn't understand
  * Tweaked from source at http://stackoverflow.com/questions/4044509/android-how-to-use-the-html-taghandler
  */
 public class WPHtmlTagHandler implements Html.TagHandler {
     private int mListItemCount = 0;
-    private Vector<String> mListParents = new Vector<String>();
+    private List<String> mListParents = new ArrayList<>();
 
     @Override
     public void handleTag(final boolean opening, final String tag, Editable output,
```

Comment:
I might be nitpicking - sorry! in this method, line 24 we are comparing `tag.equals("ul")` `OR` other similar expressions, I know this was not part of your changes per se but reviewing the code I realize there actually is a possiblity that `tag==null`, so it would be preferrable to check that `tag != null` first or, at least, change the expression to `"ul".equals(tag)`, so we are sure `.equals()` can be accessed and does not throw a NPE in such circumstance